packages/server/src/rooms.ts

```diff
@@ -33,26 +33,26 @@ export async function joinRoom(io: IOServer, socket: Socket, store: Store, roomN
 
   const player = room.players.find((player) => player.name === username);
   if (player) {
-    await doJoinRoom(socket, store, room, player);
-  } else if (room.players.length >= MAX_ROOM_SIZE) {
+    await doJoinRoom(socket, store, room);
+  } else if (room.players.length > MAX_ROOM_SIZE) {
```

Comment:
That was correct before. Let me explain. Firstly, a room is created. Then, up to two users can join. When a user joins (`joinRoom` function), we:
1. Check if that user has already joined before. If yes (first branch), we allow them in again.
2. If not, we check if the room is already full (second branch). Full here means, if two (other) users have already joined the room.
3. If it's not full (third branch), so, if 0 or 1 users have already joined, we then create the user in the "db" and also let them join the socket.io room.